source/settings/plugin/acl-plugin.rst

```diff
@@ -0,0 +1,177 @@
+.. _plugin-acl:
+
+==========
+acl plugin
+==========
+
+.. seealso:: See :ref:`acl` for a plugin overview.
+
+Settings
+^^^^^^^^
+
+.. _plugin-acl-setting_acl:
+
+``acl``
+-------
+
+- Default: <empty>
+- Values:  :ref:`string`
+
+The ACL driver to use. This setting is **REQUIRED** - if empty, the acl plugin
+is disabled.
+
+The format is:
+
+.. code-block:: none
+
+  backend[:option[:option...]]
+
+Currently, there is a single backend available: ``vfile``. This backend
+supports two ways of defining the ACL configuration:
+
+  * *global*: ACL rules are applied to all users. (v2.2.11+)
+  * *per-user*: Each user has separate ACL rules (they are stored in a
+    ``dovecot-acl`` file in each user's index directory). This is the default.
```

Comment:
They are per-folder acl files, not per-user. They are not stored in index directory, but the mailbox directory (although in obox case in CONTROL directory because there is no local mailbox directory).